scripts/sign_encrypt.py

```diff
@@ -14,6 +14,12 @@
 enc_key_type = {'SHDR_ENC_KEY_DEV_SPECIFIC': 0x0,
                 'SHDR_ENC_KEY_CLASS_WIDE': 0x1}
 
+SHDR_MAGIC = 0x4f545348
+SHDR_SIZE = 20
+
+SHDR_BOOTSTRAP_TA = 1
+SHDR_ENCRYPTED_TA = 2
```

Comment:
Nitpicking... this should be a separate commit (with the changes to lines 201-205)

Reply:
these enhancements will be on PR https://github.com/OP-TEE/optee_os/pull/5098